keras_nlp/models/f_net/f_net_presets.py

```diff
@@ -25,21 +35,22 @@
             "num_segments": 4,
         },
         "preprocessor_config": {},
+        "weights_url": "https://storage.googleapis.com/keras-nlp/models/f_net_base_en/v1/model.h5",
+        "weights_hash": "35db90842b85a985a0e54c86c00746fe",
+        "spm_proto_url": "https://storage.googleapis.com/keras-nlp/models/f_net_base_en/v1/vocab.spm",
+        "spm_proto_hash": "71c5f4610bef1daf116998a113a01f3d",
+    },
+    "f_net_large_en": {
         "metadata": {
             "description": (
-                "12-layer FNet model where case is maintained. "
+                "24-layer FNet model where case is maintained. "
                 "Trained on the C4 dataset."
             ),
-            "params": 82861056,
+            "params": 236945408,
```

Comment:
Have you used https://github.com/keras-team/keras-nlp/blob/master/tools/count_preset_params.py to count the #parameters?

Reply:
I think this is just the diff being confusing right? This isn't a new number though.